src/libraries/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -2649,12 +2649,12 @@ public abstract partial class XPathNavigator : System.Xml.XPath.XPathItem, Syste
         public virtual void AppendChild(string newChild) { }
         public virtual void AppendChild(System.Xml.XmlReader newChild) { }
         public virtual void AppendChild(System.Xml.XPath.XPathNavigator newChild) { }
-        public virtual void AppendChildElement(string prefix, string localName, string namespaceURI, string value) { }
+        public virtual void AppendChildElement(string? prefix, string localName, string? namespaceURI, string value) { }
```

Comment:
don't you also need to update https://github.com/dotnet/runtime/blob/1c27446bf70812d07e0dd62e8e808c2c5b324f14/src/libraries/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs#L401?

Reply:
Oops, missed that one. Thanks.